Rakefile

```diff
@@ -21,10 +21,9 @@ namespace :pipenv do
 end
 
 namespace :docker do
-
   desc "Build docs using docker"
   task :build do
-    exec 'docker run --rm -i -t -v "${PWD}:/doc" -u "$(id -u):$(id -g)" ohiosupercomputer/docker-sphinx make html'
+    exec 'docker run --net=host --env="DISPLAY" --volume="$HOME/.Xauthority:/root/.Xauthority:rw" --rm -i -t -v "${PWD}:/doc" -u "$(id -u):$(id -g)" ohiosupercomputer/docker-sphinx make html' 
```

Comment:
If you are exporting `DISPLAY` and bind mounting `$HOME/.Xauthority` does this require a running X server on the host?

Reply:
I believe so, yeah.

Reply:
Did `drawio-batch` require a running X server as well?

Reply:
I don't think so

Reply:
Just to let you know when I run make html on the container it seems to only work without the user option 
```
 docker run --net=host --env="DISPLAY" --volume="$HOME/.Xauthority:/root/.Xauthority:rw" --rm -i -t -v "${PWD}:/doc" ohiosupercomputer/docker-sphinx make html

```